src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+
```

Comment:
Very good usage of JavaDocs in all of your methods and classes. However, it could be even better if you had a comment explaining the purpose of that particular class. This would help the reader understand what each of the class does to get a better idea, as not everyone uses the same class name to carry out the same function. Very neat comments and javadocs though